sync/index.test.ts

```diff
@@ -226,6 +226,60 @@ test('maps input actions', async () => {
   equal(pair.rightNode.log.actions(), [{ type: 'a1' }])
 })
 
+test('calls onActions if present', async () => {
```

Comment:
I don’t see how `onActions` works in a test. I am not sure that we are property testing it.